en/deploy/README.md

```diff
@@ -130,159 +130,118 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
+$ pip3.6 install --user pythonanywhere
 ```
 
+That should print out some things like `Collecting pythonanywhere`, and eventually end with a line saying `Successfully installed (...) pythonanywhere- (...)`.
 
-### Creating a virtualenv on PythonAnywhere
-
-Just like you did on your own computer, you can create a virtualenv on PythonAnywhere. In the Bash console, type:
+Now we run the helper to automatically configure our app from GitHub. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ cd my-first-blog
-
-$ virtualenv --python=python3.6 myvenv
-Running virtualenv with interpreter /usr/bin/python3.6
-[...]
-Installing setuptools, pip...done.
-
-$ source myvenv/bin/activate
-
-(myvenv) $  pip install django~=1.11.0
-Collecting django
-[...]
-Successfully installed django-1.11.3
+$ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-blog.git
 ```
 
+As you watch that running, you'll be able to see what it's doing:
 
-> **Note** The `pip install` step can take a couple of minutes.  Patience, patience!  But if it takes more than five minutes, something is wrong.  Ask your coach.
+- Downloading your code from GitHub
+- Creating a virtualenv on PythonAnywhere, just like the one on your own PC
+- Updating your settings file with some deployment settings
+- Setting up a database on PythonAnywhere using the "migrate" command
+- Setting up your static files (we'll learn about these later)
+- And configuring PythonAnywhere to serve your web app via its API
 
-<!--TODO: think about using requirements.txt instead of pip install.-->
+On PythonAnywhere all those steps are automated, but they're the same steps you
+would have to go through with any other server provider.  The main thing to notice
+right now is that your database on PythonAnywhere is actually totally separate from
+your database on your own PC -- that means it can have different posts and admin accounts.
 
-### Creating the database on PythonAnywhere
+As a result, just as we did on your own computer, we repeat the step to
+initialize the admin account with `createsuperuser`.
 
-Here's another thing that's different between your own computer and the server: it uses a different database. So the user accounts and posts can be different on the server and on your computer.
+First, we activate the PythonAnywhere virtualenv, and then we `cd` into our
+code directory.  PythonAnywhere has named them both after your site's name:
 
-Just as we did on your own computer, we repeat the step to initialize the database on the server, with `migrate` and `createsuperuser`:
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-(mvenv) $ python manage.py migrate
-Operations to perform:
-[...]
-  Applying sessions.0001_initial... OK
-(mvenv) $ python manage.py createsuperuser
+$ workon $USER.pythonanywhere.com
```

Comment:
I can never remember which non-whitespace characters terminate a shell variable name and which don't, so when doing something like this myself, I prefer to use the `${variable_name}` syntax whenever a variable is interpolated into a larger "word" (command or argument):
```bash
workon ${USER}.pythonanywhere.com
```
But maybe that's just me, and to most of our participants this will be rather "magic" anyway (as we AFAIK don't introduce shell variables to them anywhere in the tutorial).

Reply:
I couldn't tell you about shell variable name termination either, but in any case this one works.  I'm still a bit uncomfortable about the magic here.  We could revert back to just using "yourusername.pythonanywhere.com"?  It has the benefit of teaching people that, when they copy and paste instructions from the internet, they always have to look out for any placeholder strings they need to substitute...  Happy either way tbh.

Reply:
Can we just mention that bash will substitute the username for `${USER}` (or for `$USER` if we leave it at that), and that in the pythonanywhere console, the username is the pythonanywhere account name? We can make that (more specific, but still correct) claim without introducing environment variables / shell variables _in general_.

Reply:
yes!  or at least, let's at least give it a try, then we can take a view as to whether we're throwing too many distracting explanations around...

Reply:
OK, I've done that.  I am a bit worried that we're spending a lot of time explaining some stuff that's not really core to either the deployment, or pythonanywhere.

Maybe we need to take a step back, or ask someone else to have a fresh pair of eyes on it?  

I don't know.  It'll work, and we will definitely get the benefit of avoiding copy-paste errors, but I worry that adding the `$USER` thing, and then laboriously explaining it, is one level of complication and distraction too many.  Surely attendees are used to substituting usernames by now?  Or surely that's a more important skill to teach than explaining what `$USER` is?

(in the ORM chapter for example, we expect people to substitute `User.objects.get(username='ola')` appropriately...)